benches/src/db_lookup_times_utils/utils.rs

```diff
@@ -0,0 +1,174 @@
+use crate::db_lookup_times_utils::{
+    full_block_table::{
+        BenchDatabase,
+        BenchDbColumn,
+    },
+    matrix::should_clean,
+};
+use anyhow::anyhow;
+use fuel_core::{
+    database::database_description::DatabaseDescription,
+    state::rocks_db::RocksDb,
+};
+use fuel_core_storage::kv_store::{
+    KeyValueInspect,
+    StorageColumn,
+};
+use fuel_core_types::{
+    blockchain::block::{
+        Block,
+        CompressedBlock,
+    },
+    fuel_tx::Transaction,
+    fuel_types::{
+        BlockHeight,
+        ChainId,
+    },
+};
+use itertools::Itertools;
+use rand::{
+    rngs::ThreadRng,
+    Rng,
+};
+use std::path::{
+    Path,
+    PathBuf,
+};
+use strum_macros::AsRefStr;
+
+pub type Result<T> = core::result::Result<T, anyhow::Error>;
+
+pub fn get_random_block_height(
+    rng: &mut ThreadRng,
+    block_count: BlockHeight,
+) -> BlockHeight {
+    BlockHeight::from(rng.gen_range(0..block_count.into()))
+}
+
+pub struct ConditionalTempDir {
```

Comment:
Would be nice to use ShallowTmpDir instead=)

Reply:
yup, I'm rebasing rn

Reply:
addressed in [ade2082](https://github.com/FuelLabs/fuel-core/pull/2159/commits/ade208296a97a48fd5cefdd08d6e1a2c844ddfb5)